File: API/api/core/src/main/java/org/leadisteam/leadisjourney/api/core/Tutorial.java
package org.leadisteam.leadisjourney.api.core;

public class Tutorial {

	private String id;
	private String title;
	private Source source;
	
	Tutorial(String id, String title, Source source){
		this.setId(id);
		this.setTitle(title);
		this.setSource(source);
	}

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getTitle() {
		return title;
	}

	public void setTitle(String title) {
		this.title = title;
	}

	public Source getSource() {
		return source;
	}

	public void setSource(Source source) {
		this.source = source;
	}
}

File: API/LeadisTeam.LeadisJourney.Api/src/LeadisTeam.LeadisJourney.Core/Entities/Tutorial.cs
namespace LeadisTeam.LeadisJourney.Core.Entities
{
    public class Tutorial : IEntity
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public TutorialSource Source { get; set; }
        public Exercice Exercice { get; set; }
    }
}